db/schema.rb

```diff
@@ -780,10 +780,10 @@
   end
 
   create_table "vestal_versions", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
-    t.integer  "versioned_id"
     t.string   "versioned_type"
-    t.integer  "user_id"
+    t.integer  "versioned_id"
```

Comment:
These are just moving ordering around of the columns, but can we exclude these changes?